run_selenium_system_test.sh

```diff
@@ -63,7 +63,7 @@ while [ "$1" != "" ]; do
        java_opts+=("-Denvironment=$VALUE")
        ;;
    --browser | -b)
-   if [ "$VALUE" != 'chrome' ] && [ "$VALUE" != 'firefox' ] && [ "$VALUE" != 'zap-chrome' ] && [ "$VALUE" != 'headless' ] && [ "$VALUE" != 'browserstack' ] && [ "$VALUE" != 'browserstack1' ] && [ "$VALUE" != 'browserstack2' ] && [ "$VALUE" != 'browserstack3' ] && [ "$VALUE" != 'browserstack4' ]
+   if [ "$VALUE" != 'chrome' ] && "$VALUE" != 'chrome' ] && [ "$VALUE" != 'firefox' ] && [ "$VALUE" != 'zap-chrome' ] && [ "$VALUE" != 'headless' ] && [ "$VALUE" != 'browserstack' ] && [ "$VALUE" != 'browserstack1' ] && [ "$VALUE" != 'browserstack2' ] && [ "$VALUE" != 'browserstack3' ] && [ "$VALUE" != 'browserstack4' ]
```

Comment:
This doesn't need to be 'remote-chrome' like at https://github.com/hmrc/help-to-save-frontend/pull/702/files#diff-890d98aad16848d018cb513e1691ed9fR66?

Reply:
thanks @MarcusQA  copy - paste error - fixed it